lib/generators/litestack/install/templates/database.yml

```diff
@@ -36,5 +36,5 @@ test:
 # `LITESTACK_DATA_PATH` environment variable.
 production:
   <<: *default
-  database: <%= Litesupport.root("production").join("data.sqlite3") %>
+  database: <%= Litesupport.root.join("data.sqlite3") %>
```

Comment:
Wondering if the `database` key could just be moved to default now, given it's now the same for all environments anyway?

Reply:
@wolfgangrittner Yes, could do that if the code was working properly, which it isn't. I'm not sure what I was thinking, because this code would result in test/development databases overwriting each other. Not good 😰

I think the code should be changed to ` Litesupport.root.join("development.sqlite3")`, ` Litesupport.root.join("test.sqlite3")` and so on. What do you reckon?